src/lcd.c

```diff
@@ -137,6 +139,32 @@ void myGPIOB_Init(void){
     GPIO_Init(GPIOB, &GPIO_InitStruct);
 }
 
+void LCD_CustomHzSymbol(void){
+  char customChar[8] = {
+    0b10001,
+    0b11111,
+    0b10001,
+    0b01110,
+    0b00010,
+    0b00100,
+    0b01000,
+    0b01110
+  };
+  // resets the RS and R/W pins
+  LCD_SendWord(LCD_COMMAND, 0x00);
+  // Sets CG RAM address
+  LCD_SendWord(LCD_COMMAND, 0x40);
+  // all subsequent data is written and auto incremented to CG RAM address
+  //each byte (represeting 1 line of the custom char) is written to byte locations
+  //64 to 71 of the CG RAM (Custom Generator Ram)
+  for(int a = 0; a < 8; a++){
```

Comment:
Remember my comment about declaring the index in the `for` loop to minimize scope? Turns out that some popular C project ([like this one](http://git.kernel.org/cgit/git/git.git/tree/commit.c#n558)) don't follow this idiom! I suspect it's because they're paranoid about backwards compatibility and variable declarations in loops was only added in [C99](https://en.wikipedia.org/wiki/C99) 💡